resources/public/api/conf/2.0/application.yaml

```diff
@@ -121,7 +121,9 @@ paths:
       description: >
         Get all departure declarations sent to the customs office of departure within the past 31 days.
 
-        You can filter for departure declarations updated after a specific date and time and/or submitted with a specified EORI.
+        You can filter for departure declarations updated after and/or before a specific date and time and/or submitted with a specified EORI.
+
+        For a multi-page query you can filter for a specific page along with/or not, the number of items per page.

```

Comment:
along with/or not?

Reply:
:+1:

Reply:
@derekspalmer - Would 'optionally' be a better choice than 'along with'? Also, should 'specified' be 'specific'?

Suggestion:
You can filter for departure declarations updated before and/or after a specific date and time, and optionally filter for departures with a specific EORI.